frontend/src/views/external/electrification/project/ProjectView.vue

```diff
@@ -96,207 +100,150 @@ onBeforeMount(async () => {
-            {{ t('e.common.projectView.createdBy') }}:
-            <span
-              v-if="createdBy"
-              class="font-bold"
-            >
-              {{ createdBy?.firstName }} {{ createdBy?.lastName }}
-            </span>
-            <span v-else>-</span>
-          </span>
-          <span>
-            Navigator:
-            <span
-              v-if="assignee"
-              class="font-bold"
-            >
-              {{ assignee?.firstName }} {{ assignee?.lastName }}
-            </span>
-            <span v-else>-</span>
-          </span>
-        </div>
       </div>
       <Button
         v-if="
           canNavigate(NavigationPermission.EXT_ELECTRIFICATION) &&
           getProject?.submissionType !== SubmissionType.INAPPLICABLE
         "
         class="p-button-sm header-btn mt-3"
-        label="Ask my Navigator"
-        outlined
+        :label="t('e.common.projectView.askMyNavigator')"
         @click="
           router.push({
             name: RouteName.EXT_ELECTRIFICATION_PROJECT_ENQUIRY
           })
         "
-      />
+      >
+        <AskMyNavigator />
+        {{ t('e.common.projectView.askMyNavigator') }}
+      </Button>
     </div>
 
-    <div
-      v-if="getProject?.submissionType === SubmissionType.INAPPLICABLE"
-      class="inapplicable-block p-4 mt-12"
-    >
-      {{ t('e.electrification.projectView.inapplicableSubmissionType') }}
-    </div>
-    <div>
-      <h3 class="mb-8 mt-16">{{ t('e.common.projectView.recommendedPermits') }}</h3>
-    </div>
-    <div
-      v-if="!getAuthsNeeded?.length"
-      class="empty-block p-8 mb-2"
+    <Tabs
+      :value="activeTab"
+      class="mt-3"
     >
-      {{ t('e.common.projectView.recommendedPermitsDesc') }}
-    </div>
-    <Card
-      v-for="permit in getAuthsNeeded"
-      :key="permit.permitId"
-      class="app-primary-color permit-card mb-2"
-    >
-      <template #content>
-        <h5 class="m-0 p-0">{{ permit.permitType.name }}</h5>
-      </template>
-    </Card>
-    <Accordion
-      v-if="getAuthsNotNeeded?.length"
-      class="app-primary-color"
-      :value="undefined"
-      collapse-icon="pi pi-chevron-up"
-      expand-icon="pi pi-chevron-right"
-    >
-      <AccordionPanel value="0">
-        <AccordionHeader>{{ t('e.common.projectView.notNeeded') }}</AccordionHeader>
-        <AccordionContent>
+      <TabList>
+        <Tab :value="0">
+          <font-awesome-icon
+            class="mr-2 ellipsis-icon"
+            icon="fa-solid fa-file"
+          />
+          {{ t('e.common.projectView.tabAuthorizations') }}
+        </Tab>
+        <Tab :value="1">
+          <font-awesome-icon
+            class="mr-2 ellipsis-icon"
+            icon="fa-solid fa-file-circle-question"
+          />
+          {{ t('i.common.projectView.tabRelatedEnquiries') }}
+        </Tab>
+      </TabList>
+      <TabPanels>
+        <TabPanel :value="0">
+          <BasicProjectInfo
+            :assignee="
```

Comment:
`assignee` and `created-by` props should be handed computed values based on the states.